x-pack/plugins/uptime/server/rest_api/pings/journey_screenshots.ts

```diff
@@ -18,6 +18,9 @@ export const createJourneyScreenshotRoute: UMRestApiRouteFactory = (libs: UMServ
       stepIndex: schema.number(),
       _debug: schema.maybe(schema.boolean()),
     }),
+    query: schema.object({
```

Comment:
What is this doing, for my edification?

Reply:
you can do` window.sessionStorage.setItem('uptime_debug', true)`

and it will log es queries being executed as json in kibana logs